pages/search/index.tsx

```diff
@@ -29,7 +29,9 @@ export default function Search() {
     <>
       <Head title={router.query.q && `"${router.query.q}"`} />
       <Section
-        title={`${t("search")}${router.query.q && ` "${router.query.q}"`}`}
+        title={`${algorithms.length === 0 ? t("No results for") : t("search")}${
```

Comment:
You would still need to add a new string for it to be able to be translated properly. Replace the `t("No results for")` with `t("noResults")` and add `No results for` as a new string here:
https://github.com/TheAlgorithms/website/blob/main/public/locales/en/common.json

Reply:
I have made the changes and add a message for empty query, please review.